ios/RN/RNCamera.m

```diff
@@ -1834,7 +1834,7 @@ - (void)cleanupCamera {
     }
 }
 
-- (void)mirrorVideo:(NSURL *)inputURL completion:(void (^)(NSURL* outputUR))completion {
+- (void)mirrorVideo:(NSURL *)inputURL completion:(void (^)(NSURL* outputURL))completion {
```

Comment:
won't this conflicts with

https://github.com/react-native-community/react-native-camera/pull/2625/files#diff-deff8cbb3789a329a4a17e2e3bd13bceR1866

Reply:
> won't this conflicts with
> 
> https://github.com/react-native-community/react-native-camera/pull/2625/files#diff-deff8cbb3789a329a4a17e2e3bd13bceR1866

might've been a mistake but it looks like you just linked back to this diff